hieradata/site/ls/cluster/auxtel-ccs.yaml

```diff
@@ -0,0 +1,15 @@
+---
+ccs_site: "base"
+ccs_sal::dds_interface: "auxtel-mcm-dds.ls.lsst.org"
+
+## base-teststand-alerts
+ccs_software::service_email: "base-teststand-alerts-aaaai5j4osevcaaobtog67nxlq@lsstc.slack.com"
+
+ccs_monit::alert:
+  - "base-teststand-alerts-aaaai5j4osevcaaobtog67nxlq@lsstc.slack.com"
+
+clustershell::groupmembers:
+  misc: {group: "misc", member: "auxtel-mcm,auxtel-dc01,auxtel-fp01"}
+  #hcu: {group: "hcu", member: "auxtel-hcu01"}
+  #all: {group: "all", member: "@misc,@hcu"}
+  all: {group: "all", member: "@misc"}
```

Comment:
I believe @cbarria is referring for tests of these values (also for the ones in lsstcam-ccs), which are very similar as the ones he added on 
https://github.com/lsst-it/lsst-control/pull/762. 
For the clustershell::groupmembers, you can find one as example at 
https://github.com/lsst-it/lsst-control/blob/production/spec/hosts/nodes/manke01.ls.lsst.org_spec.rb#L30-L39
btw, I would add the test for auxtel-ccs at auxtel-dc01.ls.lsst.org_spec.rb
https://github.com/lsst-it/lsst-control/blob/production/spec/hosts/nodes/auxtel-dc01.ls.lsst.org_spec.rb 
and for lsstcam-ccs at lsstcam-dc01.ls.lsst.org_spec.rb
https://github.com/lsst-it/lsst-control/blob/production/spec/hosts/nodes/lsstcam-dc01.ls.lsst.org_spec.rb

Reply:
I did try to add some tests like those, in https://github.com/lsst-it/lsst-control/pull/760/commits/6268a30482b63d830949df14e2f8caa5039a8994 
but as I said, 2/3 of them fail because the hiera settings for ccs_software::service_email and ccs_monit::alert: are apparently not being set in the test environment. Can someone explain to me how it works?

I can try to add tests for the clustershell setting too, but then surely I would just be testing that the clustershell puppet module works, which is something that should be done in that module's own tests?

Reply:
Oh, maybe now I see: lsstcam-db01.ls.lsst.org_spec.rb has no cluster setting...

Reply:
Adding the cluster setting revealed that the network tests weren't correct.

Reply:
Tests added, please re-review.